src/Blueman/Console/Command/ConvertCommand.php

```diff
@@ -39,6 +39,20 @@ protected function configure()
                 InputOption::VALUE_REQUIRED,
                 'The base host of your API (e.g. https://api.example.com/v1).'
             )
+            ->addOption(
+                'tests-filename',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'The JSON file name with Postman tests (located at --path)',
+                'blueman.tests.md'
+            )
+            ->addOption(
+                'tests-include',
+                null,
+                InputOption::VALUE_OPTIONAL,
```

Comment:
Can you make this `InputOption::VALUE_NONE` and change the option name to `include-tests` that way the command will be simpler, e.g. `$ blueman convert api.json --include-tests  --tests-filename=<YOUR_FILNAME>.md`